internal/pkg/cli/dialog/branches.go

```diff
@@ -1,4 +1,4 @@
-package interaction
+package dialog
```

Comment:
3. commit obsahuje interakciu s uzivatelom cez CLI. Balicky `dialog` a `prompt`.
![image](https://user-images.githubusercontent.com/19371734/143894491-53bb09eb-05c5-46a1-885e-5da465d1c92e.png)

Opat je to iba vykopirovany kod a testy. Takto je to oddelene od CLI a vo vlastnych balickoch.